controllers/velero.go

```diff
@@ -392,11 +392,14 @@ func (r *DPAReconciler) buildVeleroDeployment(veleroDeployment *appsv1.Deploymen
 	}
 	r.ReconcileRestoreResourcesVersionPriority(dpa)
 
+	// get resource requirements for velero deployment
+	veleroResourceReqs, _ := r.getVeleroResourceReqs(dpa)
```

Comment:
I would add a comment here that err is checked in validator.go otherwise just check err anways.

Reply:
Done, updated PR.